### ISSUE TITLE
Profile settings are not saved on the lastest appimage version

### ISSUE THREAD

**Issue by BarbzYHOOL** [OpenShot/openshot-qt#1107]
**System Details:** 
* Operating System / Distro: Ubuntu 14.04
* OpenShot Version: 2.4.1

**Issue Description and steps to reproduce:**
In 2.4.0 I had set a profile to 720p 25images per sec and now on 2.4.1 it's still set to this but it will load another one (MP4 h.264 1024x576) when exporting. Annoying

**Comment by peanutbutterandcrackers** [comment]
@BarbzYHOOL - Perhpas it has been solved now in the latest app image? Perhaps you'd also like to switch to using the [PPA](http://openshot.org/ppa) instead?

**Comment by BarbzYHOOL** [comment]
no, installation by ppa is bugged as hell, there is no file menu

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Yes, the PPA is not great from what I have heard. Personally I stick with the AppImage.

**Comment by peanutbutterandcrackers** [comment]
Strange... I use daily PPA on my Elementary OS machine, and it works perfectly fine. Previously, I used the stable ones, and it was fine too. Just the other day, I installed OpenShot on my friend's 32 bit computer running Linux Lite (the latest version), using the daily PPA and it worked great too. No such bugs. And, my brother uses a stable version of OpenShot in his Ubuntu 16.04.3 machine back at home, without any such problems, either... Perhaps, it's better now? Maybe give it a shot? o.O
